web/styles/popovers.css

```diff
@@ -1339,3 +1339,16 @@ ul.navbar-dropdown-menu-outer-list {
         display: block;
     }
 }
+
+.navbar-dropdown-hotkey-hint {
```

Comment:
I think it's possibly we'll end up renaming all the `navbar-dropdown` classes to `popover-` class names if we end up migrating other popovers to this style, but this seems fine for now as I'm not entirely certain whether such a migration is the plan.